velox/docker/adapters_build.dockerfile

```diff
@@ -47,6 +47,43 @@ ${BUILD_BASE_DIR}/${BUILD_TYPE}/_deps/nvcomp_proprietary_binary-src/lib64"
 
 WORKDIR /workspace/velox
 
+# Copy and run CentOS adapters setup script
```

Comment:
I don't think this is the right place for dependency updates. These updates should likely be in https://github.com/facebookincubator/velox/blob/6998a1461ded77060b817ff3179d3f806fd0b625/scripts/setup-centos-adapters.sh.

Reply:
Created a separate dependency image.